client/post-editor/drafts-button/index.jsx

```diff
@@ -10,12 +10,12 @@ import { translate } from 'i18n-calypso';
  */
 import Button from 'components/button';
 import Count from 'components/count';
-import QueryPostCounts from 'components/data/query-post-counts';
+import QueryPosts from 'components/data/query-posts';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import { isJetpackSite } from 'state/sites/selectors';
-import { getAllPostCount } from 'state/posts/counts/selectors';
```

Comment:
I think I'll drop the count from the link and then updates the DraftList to something like this (similar to `/posts`) 

<img width="258" alt="capture d ecran 2016-10-04 a 3 01 57 pm" src="https://cloud.githubusercontent.com/assets/272444/19077314/02f27282-8a44-11e6-82ef-db813914df35.png">

Does this look good for you ?